packages/core/src/common/_mixins.scss

```diff
@@ -91,6 +91,17 @@ $pt-dark-intent-text-colors: (
   outline: $pt-outline-color solid 2px;
   outline-offset: $offset;
   -moz-outline-radius: 6px;
+
+  // this should become default in bp6, providing new behavior as a class for now to avoid breaking changes
+  // to focus colors
+  /* stylelint-disable-next-line @blueprintjs/no-prefix-literal */
+  .bp5-focus-enhanced-contrast & {
+    outline: $pt-focus-indicator-color solid 2px;
+  }
+  /* stylelint-disable-next-line @blueprintjs/no-prefix-literal */
+  .#{$ns}-dark.bp5-focus-enhanced-contrast & {
+    outline-color: $pt-dark-focus-indicator-color;
+  }
```

Comment:
I think we should just do it. I wouldn't call this a breaking change. If we do introduce this class, you should use `#{$ns}` instead of `bp5`, which is what this lint rule is trying to tell you.